layers/+lang/alda/README.org

```diff
@@ -0,0 +1,33 @@
+#+TITLE: Alda Layer
+
+* Table of Contents
+ - [[#description][Description]]
+ - [[#install][Install]]
+ - [[#key-bindings][Key bindings]]
+   
+* Description
+Alda is a music composition language allowing music to easily be written and
+edited in a text file.
+
+This layer adds keybindings for alda-mode's functions, which allow Alda code
+to be interpreted and played by the running Alda server.
```

Comment:
Just a question - does one need to manually run Alda server in order to play the music?

Reply:
`alda-mode` will automatically start an Alda server if one is not running, and the user will have to run the command again. I'll update the README to reflect this.